distributed/cli/dask_remote.py

```diff
@@ -9,8 +9,12 @@
 @click.option('--host', type=str, default=None,
               help="IP or hostname of this server")
 @click.option('--port', type=int, default=8788, help="Remote Client Port")
-def main(host, port):
-    _remote(host, port)
+@click.option('--certfile', default=None,
```

Comment:
FWIW I prefer keeping the click code somewhat flat (rather than refactored with dependencies).  I think that this keeps things straightforward and easy for new developers to quickly understand, even if slightly repetitive.